python/ray/experimental/sgd/util.py

```diff
@@ -0,0 +1,120 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import ray
+import json
+import logging
+import time
+
+logger = logging.getLogger(__name__)
+
+
+def fetch(oids):
+    if ray.global_state.use_raylet:
+        local_sched_client = ray.worker.global_worker.local_scheduler_client
+        for o in oids:
+            ray_obj_id = ray.ObjectID(o)
+            local_sched_client.reconstruct_objects([ray_obj_id], True)
+    else:
+        for o in oids:
+            plasma_id = ray.pyarrow.plasma.ObjectID(o)
+            ray.worker.global_worker.plasma_client.fetch([plasma_id])
+
+
+def run_timeline(sess, ops, feed_dict={}, write_timeline=False, name=""):
```

Comment:
shouldn't use a mutable value for default arg

Reply:
Fixed.